world/config/skillmanager.py

```diff
@@ -0,0 +1,69 @@
+from world.abilities.ability import Ability
+
+#Create the skills database
+skill_manager = {}
+
+# hide skill
+skill_manager['hide'] = Ability()
+skill_manager['hide'].name = 'hide'
```

Comment:
I don't want to construct dicts in this manner, please use a dictionary literal.